bedrock/mozorg/templates/mozorg/mpl/2.0/faq.html

```diff
@@ -19,6 +23,9 @@ <h2 id="about-this-faq">About This FAQ</h2>
     <p>Please note that, while this FAQ is intended to be accurate and helpful, it <em>is not the license</em>, and may not cover important issues that affect you and your specific situation. As a result, reading the FAQ should not serve as a substitute for reading the license itself, or for seeking legal advice from a lawyer.</p>
     <p>If you see any errors in this FAQ, or have suggestions for further questions, please email <a href="mailto:licensing@mozilla.org">licensing@mozilla.org</a>.</p>
     <p>The MPL version 1.1 FAQ is still available <a href="{{ url('mozorg.mpl.1.1.faq') }}">here</a> but should not be used except as a reference for that license.</p>
+
+    <p class="last-updated">Last updated on January 30th, 2024
```

Comment:
Is there a way to automate this without having to refer to a Python file like what we do in `credits.html`?

Reply:
Not as far as I'm aware, at least without adding more work. This page rarely gets updated, so perhaps doing things manually is OK?